jugosack/math-magicians
# Testing

#### Testing

This branch includes:
-  Set up React Testing Library.
-  Write unit tests for the files operate.js and calculate.js using Jest.
-  Create unit tests for all React components:
- Use Jest snapshots to test the components.
- Use React Testing Library to simulate user interaction.
-  Run your full tests suite using Jest. All tests should pass.
